torch/utils/throughput_benchmark.py

```
@@ -5,7 +5,8 @@
 
 def format_time(time_us=None, time_ms=None, time_s=None):
     """Define time formatting."""
-    assert sum([time_us is not None, time_ms is not None, time_s is not None]) == 1
+    if time_us is not None or time_ms is not None or time_s is not None:
```

Hi, I would like to make sure that the condition is not inverted, as the current logic raises an error when any argument is not None instead of only when all arguments are None.

---

Changed to
```
if sum([time_us is not None, time_ms is not None, time_s is not None]) != 1:
        raise AssertionError("Expected only one of time_us, time_ms, time_s is given.")
```
Again, thanks for your correction.